cfme/tests/services/test_catalog_item.py

```diff
@@ -37,24 +41,42 @@ def dialog():
     flash.assert_success_message('Dialog "{}" was added'.format(dialog))
     yield dialog
 
+    try:
+        service_dialog.delete()
+    except NoSuchElementException:
+        logger.info('test_catalog_item: dialog yield fixture cleanup, dialog not found')
+        pass
 
-@pytest.yield_fixture(scope="function")
+
+@pytest.yield_fixture(scope="module")
 def catalog():
-    catalog = "cat_" + fauxfactory.gen_alphanumeric()
+    catalog = "test_cat_" + fauxfactory.gen_alphanumeric()
```

Comment:
I think for the context of this fixture, we're only using the catalog name to set a field in the catalog item. Not sure it would be 'easier', just different.  I'd defer to leads on this, what's a better practice for fixtures that provide supplemental configuration of things in the UI.